02_bubble/ailyntang/sort_test.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSortEdgeCases(t *testing.T) {
+	assert := assert.New(t)
+
+	assert.Equal([]int{}, bubble([]int{}), "empty array as input")
```

Comment:
technically this should be "empty slice" not "empty array".

Reply:
Thanks for the pickup. I've updated this.